hack/make/manifests/manifests.mk

```diff
@@ -1,5 +1,9 @@
 manifests/prepare-directory:
 	find $(MANIFESTS_DIR) -type f -not -name 'kustomization.yaml' -delete
 
-# Generates manifests e.g. CRD, RBAC etc, for Kubernetes and OpenShift
+## Generates manifests e.g. CRD, RBAC etc, for Kubernetes and OpenShift
 manifests: manifests/prepare-directory manifests/kubernetes manifests/openshift
+
+## Generate manifests for the branch's image tag
+manifests/branch: export MASTER_IMAGE=quay.io/dynatrace/dynatrace-operator:snapshot-${SNAPSHOT_SUFFIX}
```

Comment:
Just out of curiosity, why ?

Reply:
I am lazy

Reply:
the cloudnative tests use the manifests to deploy the operator. The image change is done in the kustomize, by default, the manifests only have the `:snapshot` tag. I did this so I can run the e2e tests more easily